cartridges/int_cybs_sfra_base/cartridge/scripts/http/payments.js

```diff
@@ -100,7 +120,12 @@ function httpAuthorizeWithToken(cardData, customerEmail, referenceInformationCod
         if (dw.system.Site.getCurrent().getCustomPreferenceValue('Cybersource_GooglePayTransactionType').value === 'sale' ){
             request.processingInformation.capture = true;
         }
-    }
+    }  
+    if (order.paymentInstruments[0].paymentMethod === 'CREDIT_CARD') {
+        if (dw.system.Site.getCurrent().getCustomPreferenceValue('Cybersource_CreditCardTransactionType').value === 'sale' ){
```

Comment:
The configuration value inside:
link_cybersource\metadata\payments_metadata\meta\Core.xml
is defined as `Cybersource_CardTransactionType` and **not**  `Cybersource_CreditCardTransactionType`